config/redis.yml

```diff
@@ -102,6 +102,9 @@ development: &defaults
   va_mobile_session_refresh_lock:
     namespace: va-mobile-session-refresh-lock
     each_ttl: 60
+  va_mobile_sm_store:
```

Comment:
This is admittedly 100% confusing, and partly my fault as I worked on VAOS before the 'flagship' mobile app existed, but the other `va_mobile_...` things here don't refer to the mobile app. They refer to the VA Mobile Framework (VAMF, of which we only use VAOS related services). IAM sessions for the that the Mobile App uses for user sessions are under `iam_...`. Although obviously since this is for SM client sessions you don't really want that either 😅

Reply:
hmm...yeah, maybe I'll flip it  to "sm_store_mobile" or something.